File: exercises/beecrowd/1096/1096.ts
// https://judge.beecrowd.com/en/problems/view/1096
function main(): void {
	for (let i = 1; i <= 9; i += 2) {
		for (let j = 7; j >= 5; j--) {
			console.log(`I=${i} J=${j}`);
		}
	}
}

main();

File: exercises/beecrowd/1096/1096.go
// https://judge.beecrowd.com/en/problems/view/1096
package main

import "fmt"

func main() {
	for i := 1; i <= 9; i += 2 {
		for j := 7; j >= 5; j-- {
			fmt.Printf("I=%d J=%d\n", i, j)
		}
	}
}
